ctapipe_io_lst/calibration.py

```diff
@@ -86,10 +86,12 @@ class LSTR0Corrections(TelescopeComponent):
     This calibrator exists in lstchain for testing and prototyping purposes.
     """
     offset = IntTelescopeParameter(
-        default_value=400,
+        default_value=0,
         help=(
-            'Define the offset of the baseline'
-            ', set to 0 to disable offset subtraction'
+            'Define the offset the offset of the baseline to be subtracted'
```

Comment:
Typo in the first line.
Also the comment could be clearer, perhaps: "Define offset to be subtracted ADDITIONALLY to the drs4 baseline corrections (default)."

Reply:
Thanks, will change

Reply:
done